src/bot/requirements.txt

```diff
@@ -9,3 +9,5 @@ multidict==5.1.0
 python-dotenv==0.17.0
 typing-extensions==3.7.4.3
 yarl==1.6.3
+discord-py-slash-command==1.1.2
```

Comment:
Is the old discord dependency still required?

Reply:
Yes, the old dependency is still needed alongside the new one.